### PR TITLE
swish activation function

### PR DESCRIPTION
**Pull request by serengil**
Google researchers announced Swish and E-Swish activation functions. They showed that this new activation functions over-perform than ReLU. Actually, these functions are variants of sigmoid. They are equal to x times sigmoid. Keras backend enables to back-propagate easily.

### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [ ] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
Thanks for the PR. This activation is already present in keras-contrib: https://github.com/keras-team/keras-contrib/blob/a2f9191dcdbd7c7bfaf06c1ac4eae27712c5cba3/keras_contrib/layers/advanced_activations.py there is no need to duplicate the code and put it again in keras-team/keras.

**Comment by serengil** [comment]
Swish appears in advanced activation functions but e-swish does not exist. Please re-consider this before closing the PR.
